.github/workflows/lint.yml

```diff
@@ -122,6 +122,7 @@ jobs:
 
     - name: make run_viewer_pytest
       run: |
+        export LD_LIBRARY_PATH=$(python3 -c "import sys, os, shiboken6; sys.stdout.write(os.path.dirname(shiboken6.__file__))")
```

Comment:
Directly specific `shiboken6` path to `LD_LIBRARY_PATH` that allow OS can find `shiboken6` during runtime.